.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Release
+on:
+  push:
+    branches: [ master ]
+    tags: ["*"]
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.1.0
+        with:
+          fetch-depth: 100
+      - name: Fetch tags
+        run: git fetch --depth=100 origin +refs/tags/*:refs/tags/*
+      - uses: olafurpg/setup-scala@v7
+        with:
+          java-version: adopt@1.11
+      - uses: olafurpg/setup-gpg@v2
```

Comment:
Why do we even need `gpg`? Is it an _external_ dependency of the `ci-release` sbt task?

Reply:
we need `gpg` to be able to sign the JARs to be able to push them to Sonatype
(since `ci-release` does all the dirty work for us, you could say it's a dependency of it, a transitive one :wink: )

Reply:
As there is JVM gpg implementation (BouncyCastle), couldn't we use another sbt plugin that doesn't assume we have a tooling installed on the system? 🤔

Reply:
[`sbt-pgp`](https://github.com/sbt/sbt-pgp) (the official plugin for signing JARs, which is what `sbt-ci-release` uses under the hood) used BouncyCastle before, but this was deprecated and now GPG is the backend. The reason was

> growing feature disparities between the pure Java Bouncy Castle implementation and GnuPG ( gpg )

Background:
 * https://discuss.lightbend.com/t/announce-sbt-pgp-2-0-0/5067
 * https://github.com/sbt/sbt-pgp/issues/140
 * https://github.com/sbt/sbt-pgp/pull/146

(For the record, I too would also prefer if (not only) sbt plugins had only JVM dependencies, be it for working with git (jgit) or pgp (BouncyCastle), but it seems that the world is moving in the opposite direction.)